addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -4849,6 +4849,19 @@ X[]
                 })
             });
         });
+        describe('undo', () => {
```

Comment:
Can you add another unit test that does : 
```
editor.execCommand('columnize', 2);
undo(editor);
redu(editor);
await insertText(editor, 'x');
```

To ensure you did not break the redo with your fix.